src/plugins/borrower-profile-exp-mixin.js

```diff
@@ -29,9 +53,107 @@ export default {
 				HOME_BP_MODAL_EXP_KEY,
 				'EXP-MP-671-Dec2024',
 			);
-			if (version) {
-				this.isBpModalEnabled = version === 'b';
+			if (version) this.isBpModalEnabled = version === 'b';
+		},
+		goToLink() {
+			this.$kvTrackEvent('borrower-profile', 'go-to-old-bp', undefined, `${this.selectedLoan?.id}`);
+			window.open(`lend/${this.selectedLoan?.id}`);
+		},
+		async loadInitialBasketItems() {
+			try {
+				const basketId = this.cookieStore.get('kvbskt');
+				if (!basketId) {
+					this.basketItems = [];
+					return;
+				}
+				const { data } = await this.apollo.query({
+					query: loanCardBasketed,
+					variables: {
+						id: 0, // dummy id since we only need basket data
+						basketId
+					},
+					fetchPolicy: 'network-only'
+				});
+				this.basketItems = data?.shop?.basket?.items?.values || [];
+			} catch (error) {
+				logFormatter(error, 'error');
+				this.basketItems = [];
 			}
-		}
+		},
+		addToBasket({ loanId, lendAmount }) {
+			if (!loanId || !lendAmount) return;
+			this.$kvTrackEvent(
+				'Lending',
+				'Add to basket',
+				'lend-button-click',
+				loanId,
+				lendAmount
+			);
+			this.isAdding = true;
+			this.apollo.mutate({
+				mutation: updateLoanReservation,
+				variables: {
+					loanId,
+					price: numeral(lendAmount).format('0.00'),
+				},
+			}).then(({ errors }) => {
+				if (errors) {
+					// Handle errors from adding to basket
+					errors.forEach(error => {
+						try {
+							this.$kvTrackEvent(
+								'Lending',
+								'Add-to-Basket',
+								`Failed: ${error.message.substring(0, 40)}...`
+							);
+							Sentry.captureMessage(`Add to Basket: ${error.message}`);
+							if (hasBasketExpired(error?.extensions?.code)) {
+								// eslint-disable-next-line max-len
+								this.$showTipMsg('There was a problem adding the loan to your basket, refreshing the page to try again.', 'error');
+								return handleInvalidBasket({
+									cookieStore: this.cookieStore,
+									loan: {
+										id: loanId,
+										price: lendAmount
+									}
+								});
+							}
+							this.$showTipMsg(error.message, 'error');
+						} catch (e) {
+							// no-op
+						}
+					});
+				} else {
+					try {
+						// track facebook add to basket
+						if (typeof window !== 'undefined' && typeof fbq === 'function') {
+							window.fbq('track', 'AddToCart', { content_category: 'Loan' });
+						}
+					} catch (e) {
+						logFormatter(e, 'error');
+					}
+					const basketId = this.cookieStore.get('kvbskt');
+					setTimeout(() => {
```

Comment:
The other usages have a comment above, `// Show modal after 1s (Defined in CSS)`, so we know later why the timeout is needed.